content/actions/creating-actions/creating-a-docker-container-action.md

```diff
@@ -93,27 +93,27 @@ This metadata defines one `who-to-greet`  input and one `time` output parameter.
 
 You can choose any base Docker image and, therefore, any language for your action. The following shell script example uses the `who-to-greet` input variable to print "Hello [who-to-greet]" in the log file.
 
-Next, the script gets the current time and sets it as an output variable that actions running later in a job can use. In order for {% data variables.product.prodname_dotcom %} to recognize output variables, you must use a workflow command in a specific syntax: `echo "::set-output name=<output name>::<value>"`. For more information, see "[Workflow commands for {% data variables.product.prodname_actions %}](/actions/reference/workflow-commands-for-github-actions#setting-an-output-parameter)."
+Next, the script gets the current time and sets it as an output variable that actions running later in a job can use. In order for {% data variables.product.prodname_dotcom %} to recognize output variables, you must use a workflow command in a specific syntax: `echo "::set-output name=<output name>::<value>"`. For more information, see "[Workflow commands for {% data variables.product.prodname_actions %}](/actions/reference/workflow-commands-for-github-actions#setting-an-output-parameter)." 
 
 1. Create a new `entrypoint.sh` file in the `hello-world-docker-action` directory.
 
-1. Make your `entrypoint.sh` file executable:
-
-  ```shell
-  chmod +x entrypoint.sh
-  ```
-
 1. Add the following code to your `entrypoint.sh` file.
 
   **entrypoint.sh**
-  ```shell
+  ```shell{:copy}
   #!/bin/sh -l
   
   echo "Hello $1"
   time=$(date)
   echo "::set-output name=time::$time"
   ```
 
+1. Make your `entrypoint.sh` file executable by running following command on your system:
+
+  ```shell{:copy}
+  chmod +x entrypoint.sh
+  ```
+  
   If `entrypoint.sh` executes without any errors, the action's status is set to `success`. You can also explicitly set exit codes in your action's code to provide an action's status. For more information, see "[Setting exit codes for actions](/actions/creating-actions/setting-exit-codes-for-actions)."
```

Comment:
This description should probably stay with the `1. Add the following code to your entrypoint.sh file.` step (i.e. the step before this one)

Reply:
Hello @lucascosti, Actually I tested with different systems and found that with some systems when the file is edited after the command has been applied, this somehow removes the executable property. It may be a problem with my system. But having this step at the last ensures that it remains executable. If there is another concern please let me know.

Reply:
👋 @avinal Thanks again for these changes! Yep, I agree that having the `chmod` step last is good! My comment was that the paragraph above was previously in the step for the `entrypoint.sh` contents (it describes what the `entrypoint.sh` code does), and should stay with that step now that it's not the last one anymore. 🙂

Reply:
Ohh, sorry I misunderstood it 😅. I see it now. Thanks